exercises/concept/high-score-board/.docs/hints.md

```diff
@@ -0,0 +1,38 @@
+# Hints
+
+## 1. Create a new high score board
+
+- Create a new object with curly brackets.
+- Write the key as a string so it can include spaces.
+- Separate key and value with a colon.
+
+## 2. Add players to a score board
+
+- Use bracket notation to add a key name that is stored in a variable (the argument).
+- Use the assignment operator `=` to set a value for the new key.
+- Return the score board parameter after you added the player.
+
+## 3. Remove players from a score board
+
+- Use the [delete operator][mdn-delete].
+- Reference the key like you have done in the task before (bracket notation).
```

Comment:
I'm sorry but I don't understand the point about not using "you" in the hints. This is the "official" example from https://exercism.lol/docs/building/tracks/concept-exercises. I used that same style.
 ```markdown
 ## General

- You need to define a [constant][constant] which should contain the [integer][integers] value specified in the recipe.

## 1. Calculate the remaining oven time in minutes

- You need to define a [method][methods] with a single parameter for the actual time so far.
```
Are the docs wrong here?

Reply:
The only reason I remove you when refering to a previous exercise or task is because they might not have done that themselves.

It's not a hard requirement, but a nice thing to do.

"You should know how to do this because you did it previously" becomes "it can be done like it has been done previously".

If the student asked for help on the previous implementation, they might go back to the person they've asked help from, without another negative pointer.

Not a blocker, feel free to ignore.

Reply:
Ah ok, I misunderstood your comments/suggestions as "remove all the 'you'".

Nevertheless I would like to choose "ignore" here because of the "prefer active over passive voice" rule that was stated somewhere (can't find the reference anymore, wasn't moved to /docs on v3 website yet). That said, I still don't have merge rights so I need to you click the button. :see_no_evil: